Makefile

```diff
@@ -63,53 +62,67 @@ prep-salt: assert-dom0
 	sudo cp dom0/* /srv/salt/
 	#sudo cp -r sd-svs-disp /srv/salt/sd  # nothing there yet...
 
-remove-sd-whonix: assert-dom0
+remove-sd-whonix: assert-dom0 ## Destroys SD Whonix VM
 	-qvm-kill sd-whonix
 	-qvm-remove -f sd-whonix
 
-remove-sd-svs-disp: assert-dom0
+remove-sd-svs-disp: assert-dom0 ## Destroys SD Submission reading VM
 	-qvm-kill sd-svs-disp
 	-qvm-remove -f sd-svs-disp
 
-remove-sd-decrypt: assert-dom0
+remove-sd-decrypt: assert-dom0 ## Destroys SD GPG decryption VM
 	-qvm-kill sd-decrypt
 	-qvm-remove -f sd-decrypt
 
-remove-sd-journalist: assert-dom0
+remove-sd-journalist: assert-dom0 ## Destroys SD Journalist VM
 	-qvm-kill sd-journalist
 	-qvm-remove -f sd-journalist
 
-remove-sd-svs: assert-dom0
+remove-sd-svs: assert-dom0 ## Destroys SD SVS VM
 	-qvm-kill sd-svs
 	-qvm-remove -f sd-svs
 
-remove-sd-gpg: assert-dom0
+remove-sd-gpg: assert-dom0 ## Destroys SD GPG keystore VM
 	-qvm-kill sd-gpg
 	-qvm-remove -f sd-gpg
 
 clean: assert-dom0 remove-sd-gpg remove-sd-svs remove-sd-journalist \
-	remove-sd-svs-disp remove-sd-decrypt remove-sd-whonix
+	remove-sd-svs-disp remove-sd-decrypt remove-sd-whonix ## Destroys all SD VMs
 	@echo "Reset all VMs"
 
-test: assert-dom0
+test: assert-dom0 ## Runs all application and integration tests
```

Comment:
Sadly does not (yet?) run the integration tests, which must be run on `sd-journalist` now.

Reply:
Thank you for careful read, @joshuathayer, I will amend the comment and push a fix.